sandboxes/heka/input/heka_tcp.lua

```diff
@@ -63,9 +73,18 @@ function process_message()
                 if s == server then
                     local client = s:accept()
                     if client then
+                        local caddr, cport = client:getpeername()
+                        if not caddr then
+                            caddr = "unknown"
+                            cport = 0
+                        end
+                        if ssl_ctx then
+                            client = ssl.wrap(client, ssl_ctx)
+                            assert(client:dohandshake())
```

Comment:
This will terminate the plugin on a timeout correct? (probably not what we want)